common/src/main/java/com/microsoft/identity/common/internal/controllers/BaseController.java

```diff
@@ -368,7 +370,7 @@ protected void addDefaultScopes(final OperationParameters operationParameters) {
         operationParameters.setScopes(requestScopes);
     }
 
-    public static AccessTokenRecord getAccessTokenRecord(@NonNull final MicrosoftStsTokenResponse tokenResponse,
+    public  AccessTokenRecord getAccessTokenRecord(@NonNull final MicrosoftStsTokenResponse tokenResponse,
```

Comment:
Can we run a formatter over this? Looks like some funny indentation happening here